cmd/olympus/actions/app.go

```diff
@@ -43,9 +41,8 @@ var (
 // App is where all routes and middleware for buffalo
 // should be defined. This is the nerve center of your
 // application.
-func App() *buffalo.App {
+func App(worker worker.Worker, storage storage.Backend, eLog eventlog.Eventlog, cacheMissesLog eventlog.Appender) *buffalo.App {
```

Comment:
does it make sense to group that into single struct and pass that.
it will make easier to uptake new changes with adding, removing params

Reply:
i know there's pros and cons of bundled params. i want to hear your opinion on this

Reply:
definitely!